src/main/java/Tasks/Task.java

```diff
@@ -0,0 +1,141 @@
+package Tasks;
+
+import java.util.Objects;
+
+public abstract class Task {
+
+    private String input;
+    private boolean done;
```

Comment:
```suggestion
    private boolean isDone;
```

boolean could be named isDone to follow the naming standards